cmd/ddev/cmd/dotddev_assets/commands/host/launch

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
 ## Description: Launch a browser with the current site
-## Usage: launch
-## Example: "ddev launch"
+## Usage: launch [path]
+## Example: "ddev launch" or "ddev launch mypath"
```

Comment:
Wouldn't this be better with `ddev launch /mypath` ?

Reply:
I think we should allow both variants to avoid errors